src/cmd/init.rs

```diff
@@ -121,7 +121,7 @@ fn populate(path: &Path, compile_sass: bool, config: &str) -> Result<()> {
     if !path.exists() {
         create_dir(path)?;
     }
-    create_file(&path.join("config.toml"), config)?;
+    create_file(&path.join("zola.toml"), config)?;
```

Comment:
I can do it later but if you have time, can you extract it in a static var to avoid hardcoding it everywhere?